selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/WorkingWithTerminalTest.java

```diff
@@ -176,34 +176,34 @@ public void shouldResizeTerminal() {
       for (String partOfContent : CHECK_MC_OPENING) {
         terminal.waitTextInFirstTerminal(partOfContent);
       }
+      terminal.waitNoTextInFirstTerminal(".dockerenv");
     } catch (TimeoutException ex) {
       // remove try-catch block after issue has been resolved
       fail("Known issue https://github.com/eclipse/che/issues/10854", ex);
     }
 
-    terminal.waitNoTextInFirstTerminal(".dockerenv");
     consoles.clickOnMaximizePanelIcon();
     loader.waitOnClosed();
 
     // check resize of the terminal
-    for (String partOfContent : CHECK_MC_OPENING) {
-      try {
+    try {
+      for (String partOfContent : CHECK_MC_OPENING) {
+
```

Comment:
From readability point it's better to add empty row after `for-statement` than before.